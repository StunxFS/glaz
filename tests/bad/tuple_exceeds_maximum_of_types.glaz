fn tuple_exceeds_maximum_of_types() (
	i32, i32, i32, i32, i32, i32, i32, i32, i32
) { }
