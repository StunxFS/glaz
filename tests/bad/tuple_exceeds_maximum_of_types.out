tests/bad/tuple_exceeds_maximum_of_types.glaz:1:38-3:2: error: tuples can have a maximum of up to 8 types
     | 
   1 - fn tuple_exceeds_maximum_of_types() (
  |  	i32, i32, i32, i32, i32, i32, i32, i32, i32
   3 - ) { }
     | 
     = help: you can use a struct instead
glaz error: could not compile `tuple_exceeds_maximum_of_types`, aborting due to previous error.
